### PR TITLE
Feature/#190-reduce-maven-log-verbosity

### LINKED ISSUE #190: Reduce maven log verbosity
When a maven build downloads data, it floods the log with messages. To avoid this, we need to add this argument to each maven call:

```
--batch-mode -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
```

This removes lines like these from the log:

```
Downloading from central: https://repo.maven.apache.org/maven2/com/exasol/error-reporting-java/0.4.0/error-reporting-java-0.4.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/exasol/error-reporting-java/0.4.0/error-reporting-java-0.4.0.pom (12 kB at 24 kB/s)
```

### PR DESCRIPTION
**Pull request by kaklakariada**
Closes #190

### CODE REVIEW

File: doc/changes/changes_1.3.3.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Project keeper maven plugin 1.3.3, released 2021-11-??
+
+Code name:
+
+## Features
+
+* #190: Reduce maven log verbosity
```
Comment (morazow) [inline doc/changes/changes_1.3.3.md:7]:
```suggestion
* #190: Reduced maven log verbosity
```

**Review by morazow — CHANGES_REQUESTED** [review]
Minor editorial change request, other than that looks good!

File: .github/workflows/test_linux_build_on_windows.yml [hunk 2]
```diff
@@ -53,6 +56,8 @@ jobs:
       - name: List files
         run: ls
       - name: Install project-keeper
-        run: mvn --batch-mode install:install-file -Dfile="project-keeper.jar" -D pomFile="pom.xml"
+        run: mvn install:install-file -Dfile="project-keeper.jar" `
+            --batch-mode `
+            -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
```
Comment (morazow) [inline .github/workflows/test_linux_build_on_windows.yml:61]: Here you have back ticks as separator, I think it is a typo.

Comment (kaklakariada) [reply .github/workflows/test_linux_build_on_windows.yml:61]: This runs on powershell which requires backticks as line continuation chars.

**Review by morazow — APPROVED** [review]

**Pull request merged**